### ISSUE TITLE
Feature request: if client first param === Number, then url should default to localhost:param

### ISSUE THREAD

**Issue by alessioalex** [juliangruber/multilevel-http#11]
Because sometimes we're too lazy to write `http://localhost:port/`

I'll work on this myself soon.

**Comment by juliangruber** [comment]
yup that'd be nice!

Btw, what are you using multilevel-http for?

**Comment by alessioalex** [comment]
I'm creating a custom NPM registry based on Node and LevelDB (and thus multilevel-http) which supports custom module replication (along with all sub-dependencies, using https://github.com/alessioalex/npm-dep-chain) and tries to match as much as it can the original NPM API (so you can use the regular `npm` commands for publishing, installing etc).

I'm now at the point where I want to scale the db using `multilevel-http`, so I want to fix everything in `multilevel-http` before I proceed.

Btw, are you interested in contributing for such a thing? (I'll open source it as soon as possible)

**Comment by alessioalex** [comment]
P.S. I'm using `level-prefix`, `multilevel-http` and `level-store` from your modules atm :D Maybe even `level-exists`. So thank you very much for creating them all, they're really useful.

**Comment by juliangruber** [comment]
sweet, so a distributed version?

**Comment by juliangruber** [comment]
if you open source I might contribute :P But otherwise I currently have waaaay to many other projects.

**Comment by alessioalex** [comment]
Yeah, I want to support multiple workers so you can scale as you want (multiple cpus, machines). I will open source it as soon as it's an alpha version (which is not that far away).

But that's not the only project I'm working on and planning to use `multilevel-http`: I'm working on a Github-like service using only Node and LevelDB (and shelling out to Git currently, but maybe will use js-git in the future).

**Comment by juliangruber** [comment]
awesome :)
